app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html

```diff
@@ -21,11 +21,21 @@
       </p>
     </div>
   </div>
-  <gce-http-load-balancer-health-check ng-repeat="healthCheck in ctrl.command.loadBalancer.healthChecks"
-                                       index="$index"
-                                       command="ctrl.command"
-                                       delete-health-check="ctrl.remove('healthChecks', $index)"
-                                       health-check="healthCheck"></gce-http-load-balancer-health-check>
+
+  <gce-health-check-selector ng-repeat="healthCheck in ctrl.command.loadBalancer.healthChecks"
+                             health-checks-by-account-and-type="ctrl.healthChecksByAccountAndType"
```

Comment:
How does this work? I don't see `healthChecksByAccountAndType` or `existingHealthCheckNames` on the controller (unless I'm missing something).